typescript/src/chain.ts

```diff
@@ -31,6 +32,19 @@ export interface Bridge {
     mainUtxo: UnspentTransactionOutput
   ): Promise<void>
 
+  /**
+   * Reveals a given deposit to the on-chain contract.
+   * @param depositTx - Deposit transaction data.
+   * @param depositOutputIndex - Index of the deposit transaction output that
+   *        funds the revealed deposit.
+   * @param deposit - Data of the revealed deposit.
```

Comment:
`@param` should be without a dot at the end.

Reply:
https://github.com/keep-network/tbtc-v2/pull/298/commits/b115e9a2112233ac19ebbc455dbba8d16471797f